common/lib/dependabot/errors.rb

```diff
@@ -160,7 +160,7 @@ def self.parser_error_details(error)
   end
 
   sig { params(error: StandardError).returns(T.nilable(T::Hash[Symbol, T.untyped])) }
-  def self.updater_error_details(error)
+  def self.updater_error_details(error) # rubocop:disable Metrics/CyclomaticComplexity,Metrics/AbcSize
```

Comment:
After defining new errors that  method `Metrics/CyclomaticComplexity` and `Metrics/AbcSize` require to be disabled. You can give me feedback if you see any performance issue.